src/Apps/Artwork/Components/Seo/SeoDataForArtwork.tsx

```diff
@@ -0,0 +1,184 @@
+import { trim } from "lodash"
+import React from "react"
+
+import { SeoDataForArtwork_artwork } from "__generated__/SeoDataForArtwork_artwork.graphql"
+import { CreativeWork } from "Components/v2/Seo/CreativeWork"
+import { Product } from "Components/v2/Seo/Product"
+import { createFragmentContainer, graphql } from "react-relay"
+import { get } from "Utils/get"
+
+const { APP_URL } = process.env
```

Comment:
This should actually come from Sharify (that's our way of sharing those configs between Force and Reaction components), vs using `process.env`.

For storybooks, we have a [custom webpack plugin](https://github.com/artsy/reaction/blob/86e9ac1793333e1f9edec4b54815bec04ae9a219/.storybook/webpack.config.js) that will load values from `process.env` _into_ sharify (so that it's consistent and we can always just use sharify in components).